test/specs/scale.time.tests.js

```diff
@@ -700,10 +701,9 @@ describe('Time scale tests', function() {
 		it('should get the correct labels for ticks', function() {
 			var scale = this.scale;
 
-			expect(scale._ticks.map(function(tick) {
-				return tick.major;
-			})).toEqual([true, false, false, false, false, false, true]);
-			expect(scale.ticks).toEqual(['<8:00:00>', '<8:00:10>', '<8:00:20>', '<8:00:30>', '<8:00:40>', '<8:00:50>', '<8:01:00>']);
+			expect(scale.ticks.length).toEqual(61);
```

Comment:
Seems like a pretty big change of ticks here. 7 -> 61. Seems to indicate that maybe something is happening that shouldn't, because I don't think 61 ticks would actually render correctly here

Reply:
This doesn't actually look any different if you render the chart because `autoSkip` will skip the additional ticks. This is an artifact of the test code checking all the built ticks and not just the ones that are left after `autoSkip` has run. I'd tried to explain this in the PR description because I expected it'd raise questions, so not sure if you caught that or understood what I was saying.

Reply:
Gotcha. That’s my fault for skimming the PR details